src/sentry/tasks/auto_source_code_config.py

```diff
@@ -85,27 +85,32 @@ def process_error(error: ApiError, extra: dict[str, str]) -> None:
     default_retry_delay=60 * 10,
     max_retries=3,
 )
-def auto_source_code_config(project_id: int, event_id: str, **kwargs: Any) -> None:
+def auto_source_code_config(project_id: int, group_id: int, event_id: str, **kwargs: Any) -> None:
```

Comment:
This has to be an optional param at first, otherwise in flight tasks won't be passing it and will fail during deploy